### PR TITLE
Section restructuring

### PR DESCRIPTION
**Pull request by nedmsmith**
Created a section for acs augmentation and put ACS requirements, and phase 2 - 4 sections inside. order of triple processing became part of ACS requirements and augmentation using CoMID triples became introduction to the new ACS augmentation section

This PR addresses issues #320 and #324

Note that PR #323 overlaps this PR possibly requiring manual merge.

### CODE REVIEW

File: draft-ietf-rats-corim.md [hunk 1]
```diff
@@ -1998,14 +1998,103 @@ If the Evidence does not have a value for the mandatory `ae` fields, the Verifie
...
+measurement values are not equivalent, then a Verifier SHALL report
+an error and stop validation processing.
+
+##### Ordering of triple processing
+
+Triples interface with the ACS by either adding new ACS entries or by matching existing ACS entries before updating the ACS.
+Most triples use an `environment-map` field to select the ACS entries to match or modify.
+This field may be contained in an explicit matching condition, such as `stateful-environment-record`.
+
+The order of triples processing is important.
```
Comment (deeglaze) [inline draft-ietf-rats-corim.md:2076]: The order should not be important so long as additions are followed by triple processing that has a condition that could match it. We don’t want to impose other order dependencies unless you need to stratify triple processing more explicitly.

Given that we’re defining a processing order for base triples, it should also be allowable for profiles to specify when their triples get processed if indeed order is important. Will they always be after the base triples, or can they be specified to happen at any time?

File: draft-ietf-rats-corim.md [hunk 2]
```diff
@@ -1998,14 +1998,103 @@ If the Evidence does not have a value for the mandatory `ae` fields, the Verifie
...
+Processing a triple may result in ACS modifications that affect matching behavior of other triples.
+
+The Verifier MUST ensure that a triple including a matching condition is processed after any other triple that modifies or adds an ACS entry with an `environment-map` that is in the matching condition.
+
+This can be acheived by sorting the triples before processing, by repeating processing of some triples after ACS modifications or by other algorithms.
+
+#### ACS Augmentation Requirements {#sec-acs-aug-req}
+
+The ordering of ECTs in the ACS is not significant.
+Logically, new ECT entries are appended to the existing ACS.
```
Comment (deeglaze) [inline draft-ietf-rats-corim.md:2086]:
```suggestion
Logically, the ACS represents the conjunction of all claims, so adding an ECT entry to the existing ACS at the end is equivalent to inserting it anywhere else.
```

File: draft-ietf-rats-corim.md [hunk 3]
```diff
@@ -1998,14 +1998,103 @@ If the Evidence does not have a value for the mandatory `ae` fields, the Verifie
...
+But implementations may optimize ECT order to achieve better performance.
```
Comment (deeglaze) [inline draft-ietf-rats-corim.md:2087]:
```suggestion
Implementations may optimize ECT order to achieve better performance.
```

**Comment by yogeshbdeshpande** [comment]
Ignore my earlier comment, I think i got the hang of the PR now!

**Comment by nedmsmith** [comment]
This PR isn't trying to make content changes to sections. Merely moving sections around. There is still more work to do to get the content right.

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Review by henkbirkholz — APPROVED** [review]

File: draft-ietf-rats-corim.md [hunk 4]
```diff
@@ -1998,14 +1998,103 @@ If the Evidence does not have a value for the mandatory `ae` fields, the Verifie
...
+asserts that the actual state of one or more Claims within the
+Target Environment, as identified by `environment-map`, have the
+measurement values in `measurement-map`/`mval`.
+
+ECT authority is represented by cryptographic keys. Authority
+is asserted by digitally signing a Claim using the key. Hence, Claims are
+added to the ACS under the authority of a cryptographic key.
+
+Each Claim is encoded as an ECT. The `environment-map` and a
+key within `measurement-values-map` encode the name of the Claim.
```
[the remark this answers is not in the document]
Comment (yogeshbdeshpande) [reply draft-ietf-rats-corim.md:2043]: refer issue #334

File: draft-ietf-rats-corim.md (hunk 1 above)
Comment (yogeshbdeshpande) [reply draft-ietf-rats-corim.md:2076]: refer to issue #335

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Review by deeglaze — APPROVED** [review]

File: draft-ietf-rats-corim.md [hunk 5]
```diff
@@ -1998,14 +1998,103 @@ If the Evidence does not have a value for the mandatory `ae` fields, the Verifie
...
+
+[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/232
+
+#### ACS Processing Requirements
+
+The ACS contains the actual state of Attester's Target Environments (TEs).
+The `state-triples` field contains Evidence (from Attesters) and Endorsements
+(e.g. from `endorsed-triple-record`).
+
+CoMID Reference Values will be matched against the ACS, as per
```
[the remark this answers is not in the document]
Comment (deeglaze) [reply draft-ietf-rats-corim.md:2026]: Issue #336

**Pull request merged**